src/ng/compile.js

```diff
@@ -303,16 +303,15 @@
  *   name. Given `<my-component my-attr="parentModel">` and the isolate scope definition `scope: {
  *   localModel: '=myAttr' }`, the property `localModel` on the directive's scope will reflect the
  *   value of `parentModel` on the parent scope. Changes to `parentModel` will be reflected in
- *   `localModel` and vice versa. Optional attributes should be marked as such with a question mark:
- *   `=?` or `=?attr`. If the binding expression is non-assignable, or if the attribute isn't
```

Comment:
It is correct: http://plnkr.co/edit/7x6kyfbpWIFCFgTmZb2Z?p=preview
As mentioned below, it will only throw _"upon discovering changes to the local value, since it will be impossible to sync
   *   them back to the parent scope"_